enemy_tools.py

```diff
@@ -0,0 +1,34 @@
+import enemy
+import utils
+
+def get_enemy_size_from_type(enemy_type):
+    return utils.get_enemy_size(enemy_type)
+    
+
+# Since the stage is what determines the enemy's type, let's isolate that into a simple function
+def get_enemy_type_from_stage(stage):
+    match stage:
+        case 1:
+            return 'enemy1'
+        case 2:
+            return 'enemy2'
+        case 3:
+            return 'boss3'
+        case 4:
+            return 'boss4'
+        case 5:
+            return 'boss5'
+    
+
+# Given the stage, returns the number of enemies to spawn
+def get_enemies_for_stage(stage):
+    enemies = []
+    if stage in [1,2]:
+        for _ in range(10):
+            enemies.append(enemy.Enemy(get_enemy_type_from_stage(stage), stage >=3, stage))
+    if stage > 2:
+        for _ in range(1):
+            enemies.append(enemy.Enemy(get_enemy_type_from_stage(stage), stage >=3, stage))
+    return enemies
```

Comment:
I noticed that the stage is the thing that dictates:
- how many enemies get created
- what type of enemies get created

This function handles that logic. Notice on L28 and L31 that we do `enemy.Enemy()`. THAT is where we actually "instantiate" the blueprint from enemy.py to create ACTUAL enemies that we can then do things like `enemy.x` and `enemy.whatever()` on. 

Each instance is like its own SEPARATE "bag of variables". Just like a dict. But a little cleaner.

This is a good spot to spend a lot of time on - re-read this snippet a bunch of types and trace the code from here